src/main/java/kostka/moviecatalog/controller/MovieDetailController.java

```diff
@@ -18,18 +20,23 @@
 @Controller
 @RequestMapping("/movies/detail")
 public class MovieDetailController {
+    public static final String MOVIE_DETAIL_ATTR = "movieDetail";
```

Comment:
I would say for readability sake it would be good to group those string constants together before services (not before and after services).

Reply:
valid one :)